### PR TITLE
sets up the server side

### PR DESCRIPTION
**Pull request by Taqdeer**

### CODE REVIEW

**Review by srujanr40 — APPROVED** [review]

File: .DS_Store
Comment (rkelly123) [inline .DS_Store:1]: What file is this? I don't recognize it

File: server/app.js
Comment (rkelly123) [inline server/app.js:1]: should you add cors to the app?

File: server/public/index.html
Comment (rkelly123) [inline server/public/index.html:1]: Are these public files necessary? (including the ones other that this arbitrary file I'm commenting on)

**Review by rkelly123 — COMMENTED** [review]
Just a few simple comments

**Comment by rkelly123** [comment]
I don't see anything wrong with it, but if you haven't already, could you do a clean clone of the repo and make sure everything still works? All looks good to me, but sometimes messing with file paths has adverse effects, so just want to be on the safer side.

File: .DS_Store
Comment (Taqdeer) [reply .DS_Store:1]: added to git_ignore

File: server/app.js
Comment (Taqdeer) [reply server/app.js:1]: i will add the cors when i will start implementing the filter

File: server/public/index.html
Comment (Taqdeer) [reply server/public/index.html:1]: There are only two, one html and one css - not too helpful but gives you the idea that express is indeed running on a port as it instantly loads the html file on browser. Will eventually get rid of it.

**Pull request merged**